<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OwnerQuery</class>
 <widget class="QWidget" name="OwnerQuery">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1360</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>业主信息查询</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="nameLabel">
     <property name="text">
      <string>姓名:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="nameEdit"/>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="phoneLabel">
     <property name="text">
      <string>电话:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLineEdit" name="phoneEdit"/>
   </item>
   <item row="0" column="4">
    <widget class="QLabel" name="idNumberLabel">
     <property name="text">
      <string>身份证号:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QLineEdit" name="idNumberEdit"/>
   </item>
   <item row="0" column="6">
    <widget class="QPushButton" name="searchButton">
     <property name="text">
      <string>查询</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="7">
    <widget class="QTableWidget" name="tableWidget">
     <property name="columnCount">
      <number>7</number>
     </property>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
